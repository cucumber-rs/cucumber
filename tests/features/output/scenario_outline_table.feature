Feature: Outline

  Scenario Outline: foo
    Given foo is <bar1>
      | key | value  |
      | 1   | <bar1> |
      | 2   | <bar2> |
    When foo is <bar2>
    Then foo is <bar3>

    Examples:
      | bar1 | bar2 | bar3 |
      |  0   |  1   |  2   |
      |  3   |  4   |  5   |
