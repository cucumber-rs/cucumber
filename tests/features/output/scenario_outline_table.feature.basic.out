Feature: Outline
  Scenario Outline: foo
   ✔  Given foo is 0
       | key | value |
       | 1   | 0     |
       | 2   | 1     |
   ✔  When foo is 1
   ✔  Then foo is 2
  Scenario Outline: foo
   ✘  Given foo is 3
       | key | value |
       | 1   | 3     |
       | 2   | 4     |
      Step failed:
      Defined: tests/features/output/scenario_outline_table.feature:4:5
      Matched: tests/output.rs:11:1
      Step panicked. Captured output: assertion failed: `(left == right)`
        left: `0`,
       right: `3`
